modules/migration-converting-the-storage-class-of-persistent-volumes-cam.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/advanced-migration-options-mtc.adoc
+
+:_content-type: PROCEDURE
+[id="migration-converting-the-storage-class-of-persistent-volumes-cam_{context}"]
+= Converting the storage class of persistent volumes in the {mtc-short} web console
```

Comment:
Suggest changing title to "Converting storage classes".

If a CLI procedure exists, it should be documented.